Tests/js/views/assets/ez-dashboardblocksview-tests.js

```diff
@@ -78,7 +76,7 @@ YUI.add('ez-dashboardblocksview-tests', function (Y) {
             blocks = view.get('blocks');
 
             Y.Assert.areSame(view, block1.getTargets()[0], 'The dashboard view should be a target of a block');
-            Y.Assert.areSame(1, blocks.length, 'There should be 1 block view instance available');
+            Y.Assert.areSame(2, blocks.length, 'There should be 2 block view instances available');
```

Comment:
to avoid the need to do the very same changes when adding a new default block, I would make sure the `blocks` attribute is empty (a `this.view._set('blocks', []);` call in `setUp` is perfectly acceptable in that case).